mmcv/ops/csrc/onnxruntime/cpu/roi_align_rotated.cpp

```diff
@@ -2,6 +2,7 @@
 // https://github.com/facebookresearch/detectron2/tree/master/detectron2/layers/csrc/ROIAlignRotated
 // Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
 #include "roi_align_rotated.h"
+
```

Comment:
unintended change

Reply:
This is simply to fix the lint issue in the master branch.